packages/cli/src/workflows/workflows.controller.ts

```diff
@@ -529,7 +529,7 @@ export class WorkflowsController {
 	}
 
 	@Post('/:workflowId/activate')
-	@ProjectScope('workflow:update')
+	@ProjectScope('workflow:publish')
```

Comment:
This is the key change that switches from checking `workflow:update` to `workflow:publish` for activating/deactivating a workflow.